plugins/catalog/src/components/CatalogTable/columns.tsx

```diff
@@ -100,11 +100,17 @@ export const columnFactories = Object.freeze({
       ),
     };
   },
-  createSpecTypeColumn(): TableColumn<CatalogTableRow> {
+  createSpecTypeColumn(
+    {
+      hidden,
+    }: {
+      hidden: boolean;
+    } = { hidden: false },
```

Comment:
should default be `true` as it was before?

Reply:
yeah this was tricky. The default wasn't actually `true` because of this hack https://github.com/backstage/backstage/pull/21202/files#diff-2b7307fea1e89accb79f0f1bd27370bd5c0897566e2b3d85c76e99a841677f0eL233

which basically flip it to `false` in the render method, which shouldn't be allowed